### PR TITLE
1.19 Port

### PR DESCRIPTION
**Pull request by Diliard**
Ported the mod to 1.19 and updated the example mod.

### CODE REVIEW

File: panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/package-info.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
+ * PanelStudio-MC utility functions designed for FabricMC 1.17.x and 1.18.x.
+ * This library only depends on the Fabric Yarn mappings and not on the Fabric Mod Loader or the Fabric API.
+ * @author lukflug
+ */
```
Comment (lukflug) [inline panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/package-info.java]: Update description. Plus, add yourself as the author of this port. As well as in the classes you modified.

File: panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/package-info.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+package com.lukflug.panelstudio.mc17;
```
Comment (lukflug) [inline panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/package-info.java]: Change package name to `mc19`.

File: panelstudio-mc19/gradle.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+coreVersion = 0.2.1
+
+loom17Version = 0.12.1
+loom19Version = 1.0-SNAPSHOT
+
+minecraft17 = 1.17.1
+minecraft19 = 1.19.2
+
+mappings17 = 1.17.1+build.65
+mappings19 = 1.19.2+build.25
```
Comment (lukflug) [inline panelstudio-mc19/gradle.properties]: I don't see any need for keeping the 1.17 variables.

File: panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/MinecraftGUI.java [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+		
+		@Override
+		public boolean getModifier (int modifier) {
+			return switch (modifier) {
+				case SHIFT -> (modifiers & GLFW.GLFW_MOD_SHIFT) != 0;
+				case CTRL -> (modifiers & GLFW.GLFW_MOD_CONTROL) != 0;
+				case ALT -> (modifiers & GLFW.GLFW_MOD_ALT) != 0;
+				case SUPER -> (modifiers & GLFW.GLFW_MOD_SUPER) != 0;
+				default -> false;
+			};
```
Comment (lukflug) [inline panelstudio-mc19/src/main/java/com/lukflug/panelstudio/mc17/MinecraftGUI.java]: Please keep old C-style switch cases for consistency with the rest of the codebase.

**Review by lukflug — COMMENTED** [review]
Thank you for submitting this PR! I have gone through the changes and they mostly look good to me. I noted a few minor things that need fixing in the comments. What is also missing is appropriate changes to the GitHub CI workflows. However, I can do that myself.

**Comment by Diliard** [comment]
Updated the requested changes! Sorry for the mistakes I forgot to push my local version which fixed the variables and the description :'

Sadly I have never worked with the github CI which is why I can't help with that. But if I can find a quick guide/I get a quick explanation I would not mind changing the CI workflows.

**Comment by Diliard** [comment]
I think I figured it out :' hope the CI update works. it failed because the MC1.19 mod isnt available to compile so I think it will work once its in the repo

**Comment by lukflug** [comment]
Thank you for your changes! I'll merge it, and test and fix some things before doing a release.

**Pull request merged**